### PR TITLE
Adding few ESXi samples

### PR DESCRIPTION
**Pull request by lamw**
- service_manager_esxtop_in_vc.py
- service_manager_vscsistats_in_vc.py
- update_esxi_advanced_settings.py

### CODE REVIEW

File: samples/service_manager_esxtop_in_vc.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+import atexit
+import argparse
+import getpass
+import ssl
+
+from pyVmomi import vim
+from pyVim.connect import SmartConnect, Disconnect
+
+"""
+Example of fonnecting to the ESXTOP service provided
```
Comment (prziborowski) [inline samples/service_manager_esxtop_in_vc.py]: Can you address the typo 'fonnecting'

File: samples/service_manager_esxtop_in_vc.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                      port=int(args.port),
+                      sslContext=ssl._create_unverified_context())
+
+    atexit.register(Disconnect, si)
+
+    # "vmware.host." prefix is required when connecting to VC
+    location = "vmware.host." + args.esxi
+
+    services = si.content.serviceManager.QueryServiceList(
+        location=[location])
```
Comment (prziborowski) [inline samples/service_manager_esxtop_in_vc.py]: Does it make sense to pass in the serviceName as a parameter here to reduce the results you'd have to iterate over?

Comment (lamw) [inline samples/service_manager_esxtop_in_vc.py]: We actually only have 2 hardcoded ones, so I figure it would be simple to just keep it as is :)

**Comment by tianhao64** [comment]
@lamw Can you fix the format issue?

**Comment by lamw** [comment]
@tianhao64 sorry, which format issues?

**Comment by tianhao64** [comment]
There seems to be a line length issue:
/home/travis/build/vmware/pyvmomi-community-samples/samples/update_esxi_advanced_settings.py:103:80: E501 line too long (92 > 79 characters)

**Comment by lamw** [comment]
Do you have a suggestion on how to fix that? I wasn't able to figure out how to satisfy pep8 w/o breaking the sample itself? Do I need to just use a variable to store **ssl._create_unverified_context()**?

**Comment by prziborowski** [comment]
I believe you can either use SmartConnectNoSSL, which won't require passing in the sslContext.
You can also do:
from pyVim.connect import SmartConnectNoSSL
and that should save some width space when you use it.

**Comment by lamw** [comment]
@prziborowski Thanks! I didn't know there was a NoSSL version of SmartConnect

**Pull request merged**